SA-Policy.md

```diff
@@ -20,4 +20,83 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
+  
+Once development is complete, a team member submits the code to the version control system as a "pull request", where at least one other team member further reviews it before merging it into the code base.  New features are deployed into our staging area where they undergo further security review and stakeholder acceptance testing, as well as automated acceptance tests.
+The System Owner is responsible for ensuring appropriate staffing for security needs. The Cloud Operations team implements, configures, and maintains security controls.
+
+GSA Information Security supports and monitors the cloud.gov team, including the ISSO who serves as a liaison between GSA Information Security and the cloud.gov team.
+
+The cloud.gov 3PAO provides third-party verification and assessment of cloud.gov security.
+
+As part of cloud.gov account management and access control, each service has a list of privileged accounts with the identities of the cloud.gov team members who have those privileges. See AC and IA control families for more details.
+
+See SA-3.
+
+The cloud.gov program utilizes agile development processes. Changes are made early, often, and iteratively. Functions, ports, and protocols are part of this process. All such configuration is captured in version control and deployed in a fully automated process. Due the complete virtualization of cloud.gov in the AWS GovCloud, and the almost complete automation of the system, these changes can be made immediately and safely as requirements evolve.
+
+See SA-4 (9).
+
+cloud.gov itself does not accept personal identity verification (PIV) cards. Customers may configure an enterprise identity provider that directly accepts PIV cards, either as a primary authenticator or a multi-factor authenticator (MFA).
+
+See SA-4 (10).
+
+Cloud Operations always obtains complete documentation, including administrator and user documentation, for any technology that is used within cloud.gov. The maintenance of administrator- and user- facing documentation, in the form of version-controlled changes in a code repository or our documentation repository, is an assumed requirement for all changes. Currently, cloud.gov only uses technology whose documentation can be shared publicly.
+18F values transparency and collaboration. All documentation for technologies used by cloud.gov is either linked to directly from https://docs.cloud.gov, or is shared broadly within GSA via Google Apps for Government.
+
+See SA-5. 
+
+cloud.gov applies security best practices, including but not limited to:
+1.	Representing the entire system as “code”, so all changes and side effects can be quickly identified
```

Comment:
This list doesn't show up as a numbered list in the rendered markdown, but eh, it's fine.